src/components/feedback-widget/feedback-widget.config.yml

```diff
@@ -0,0 +1,16 @@
+label: Feedback widget
+default: success
+
+context:
+  widgetVersion: "0.7.0"
+  feedbackApiUrl: "https://innovation.nj.gov/app/feedback/dev"
+  mockResponse: { status: 200, body: {message: "Success", feedbackId: 1} }
+  mockResponseDelayInMs: 1000
+  
+variants:
+  - name: success
+    label: Success
+  - name: server error
+    label: API error (500 status)
+    context: 
+      mockResponse: { status: 500, body: {message: "example server error" } }
```

Comment:
[nit-picky] Can we add a newline here, so we don't have get the `no newline symbol`?